template/lambda-sqs-worker-cdk/src/app.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-console */
+import { SQSEvent, SQSHandler } from 'aws-lambda';
+
+export const handler: SQSHandler = (event: SQSEvent) => {
+  console.log(event);
```

Comment:
This could mislead users into thinking we advocate the use of `console.log` on production applications. We should

1. Bundle `@seek/logger` at minimum and use that
2. Avoid including code like this that logs too much from both cost and potential PII angles